src/color.rs

```diff
@@ -386,47 +387,84 @@ fn clamp_f32(val: f32) -> u8 {
 
 #[inline]
 fn parse_color_function(name: &str, arguments: &mut Parser) -> Result<Color, ()> {
-    let (is_rgb, has_alpha) = match_ignore_ascii_case! { name,
-        "rgba" => (true, true),
-        "rgb" => (true, false),
-        "hsl" => (false, false),
-        "hsla" => (false, true),
+    let is_rgb = match_ignore_ascii_case! { name,
+        "rgba" => true,
```

Comment:
I believe `match_ignore_ascii_case` now supports more general patterns, so you can write it down as:

```rust
let is_rgb = match_ignore_ascii_case! { name,
    "rgba" | "rgb" => true,
    "hsl" | "hsla" => false,
    _ => return Err(()),
};
```